src/main/java/teammates/storage/entity/FeedbackQuestion.java

```diff
@@ -51,6 +51,9 @@
     private int questionNumber;
     
     @Persistent
+    private Boolean isQuestionCompulsory;
```

Comment:
Oh sorry I forget this. Actually I was thinking of updating this in the next commit.